lib/Driver/ToolChains/Flang.cpp

```diff
@@ -53,6 +54,8 @@ void FlangFrontend::ConstructJob(Compilation &C, const JobAction &JA,
   // Check number of inputs for sanity. We need at least one input.
   assert(Inputs.size() >= 1 && "Must have at least one input.");
 
+  bool IsOpenMPDevice = JA.isDeviceOffloading(Action::OFK_OpenMP);
+
```

Comment:
Similarly, can't this be moved much closer to where it is used? Is the variable even needed? It looks like it is only used once.